jenkins/run-server-func-tests

```diff
@@ -18,15 +18,20 @@ fi
 function cleanup {
     # Remove the pod which we just created and ran; remove any dangling
     # containers; and then remove any dangling images.
+    echo "Forcefully removing the Pbench Server container ..." >&2
     podman rm --force --ignore ${1}
+    echo "Forcefully removing the Pbench Services container ..." >&2
```

Comment:
s/container/pod/ ?  (This would also match lines 89 & 96....)

Reply:
Fixed.